### DISCUSSION TITLE
Number of digits counting

### DISCUSSION THREAD

**Discussion by dietmarwo** [esa/SpOC3#13] [Challenge 2: Interferometric Mission]
How many digits are relevant for the rank? More than are  shown?
https://optimise.esa.int/challenge/spoc-3-interferometric-mission/p/large-observatory-in-halo 
Here we currently have theEs ranked third, but they didn't submit 1.559 first. 

Ok, they improved in the meantime, so the question is outdated

**Comment by maxbannach** [comment]
Anyhow, the display is indeed truncated. If the order does not correlate with submission time, the better solution is actually better.
